src/veh_interact.cpp

```diff
@@ -1168,9 +1169,16 @@ bool veh_interact::overview( std::function<bool( const vehicle_part &pt )> enabl
         if( pt.is_tank() && !pt.is_broken() ) {
             auto details = []( const vehicle_part & pt, const catacurses::window & w, int y ) {
                 if( pt.ammo_current() != "null" ) {
+                    std::string specials = "";
```

Comment:
`std::string` is automatically initialized to an empty string, you don't need to do this explicitly.